### PR TITLE
Adding add user to org Api

### PR DESCRIPTION
**Pull request by sknirmalkar89**

### CODE REVIEW

File: src/service/programService.js [hunk 1]
```diff
@@ -738,7 +738,123 @@ function createUserRecords(user, userOrgMapDetails, orgInfoList, callback) {
...
+    body: {
+      id: "open-saber.registry.create",
+      request: {
+        User: reqObj.User
+      }
+    }
+  }
+
+  registryService.addRecord(addUserReq, (err, res) => {
+    if (res) {
```
Comment (pallakartheekreddy) [inline src/service/programService.js]: Why nested if-else we can handle all this in one if else.

File: src/service/programService.js [hunk 2]
```diff
@@ -738,7 +738,123 @@ function createUserRecords(user, userOrgMapDetails, orgInfoList, callback) {
...
+        callback("Encounted some error while adding user")
+      }
+    }
+  });
+
+}
+
+function mapUserToOrg(reqObj,userObj, callback) {
+
+  let userOrgObj ={
```
Comment (pallakartheekreddy) [inline src/service/programService.js]: merge two objects into one

File: src/service/programService.js [hunk 3]
```diff
@@ -738,7 +738,123 @@ function createUserRecords(user, userOrgMapDetails, orgInfoList, callback) {
...
+    body: {
+      id: "open-saber.registry.create",
+      request: {
+        User_Org:userOrgObj
+      }
+    }
+  }
+
+  registryService.addRecord(mapReq, (err, res) => {
+    if (res) {
```
Comment (pallakartheekreddy) [inline src/service/programService.js]: here also remove nested if-else

File: src/service/programService.js (hunk 2 above)
Comment (sknirmalkar89) [reply src/service/programService.js]: changes done

File: src/service/programService.js (hunk 3 above)
Comment (sknirmalkar89) [reply src/service/programService.js]: changes done

**Pull request closed without merging**